guides/v2.1/frontend-dev-guide/themes/theme-structure.md

```diff
@@ -152,7 +152,7 @@ The directories and files structure described below is the most extended one. It
       </td>
       <td colspan="1">required</td>
       <td colspan="1">
-        This directory contains a theme preview (a screenshot of your theme).
+        This directory contains a theme preview (a screenshot of your theme). It should not be required as admin theme do not have it and also in [Create a new storefront theme page](https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/themes/theme-create.html#fedg_create_theme_how-to_declare).
```

Comment:
If this isn't required, then we should simply change "required" to "optional" in the preceding cell and remove this text.